src/lib/utilities/API/api.ts

```diff
@@ -12,35 +14,59 @@ const instance = axios.create({
   baseURL: BASE_URL,
 });
 
-type GetConfigType = {
+export type GetConfigType = {
   params: {
     page?: number;
     size?: number;
     sort?: string[];
+    direction?: number;
+    type?: number;
+    tags?: string;
   };
 };
 
-type GetTagConfigType = {
+export type GetExpertsConfigType = GetConfigType & {
+  params: { directions?: number[] };
+};
+
+export type GetTagConfigType = {
   params: {
     value: string;
     limit?: number;
   };
 };
 
-type PostTagRequestBodyType = {
+export type PostTagRequestBodyType = {
   tag: string;
 };
 
-type PostsTypeType = 'important' | 'latest';
+export type GeneralGetType =
+  | GetConfigType
+  | GetTagConfigType
+  | GetExpertsConfigType;
+
+const defaultConfig = {
+  paramsSerializer: (params: GeneralGetType): string => {
```

Comment:
don't specify a concrete type here.
`{ [key: string]: any }` should be enough

Reply:
fixed